MSCI445-Lab2/EmailLab/MainWindow.xaml.cs

```diff
@@ -39,21 +40,47 @@ public MainWindow()
 
 		private void Submit_Button_Click(object sender, RoutedEventArgs e)
         {
-			validateInput();
 			myEmail = new MyEmail(SUBJECT_TEXTBOX.Text, 
 				MESSAGE_TEXTBOX.Text, FROM_TEXTBOX.Text, "", TO_TEXTBOX.Text);
 			var passwordWindow = new PasswordWindow(myEmail);
-			passwordWindow.Show();
+			if (this.validateInput() == true) { passwordWindow.Show(); };
```

Comment:
can we move this line to before `myEmail` ? The reason is we will do the `sendEmail` method in `MyEmail` class, so we should check the inputs before initializing `myEmail`

Reply:
We can pass `SUBJECT_TEXTBox.Text`, `MESSAGE_TEXTBOX.Text`... into this method as parameters